changelogs/fragments/homebrew-cask-at-symbol-fix.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - homebrew_cask - fixed issue where a cask with `@` in the name is incorrectly reported as invalid (https://github.com/ansible-collections/community.general/issues/733)
```

```suggestion
  - homebrew_cask - fixed issue where a cask with `@` in the name is incorrectly reported as invalid (https://github.com/ansible-collections/community.general/issues/733).
```

---

Also use double backticks:
```suggestion
  - homebrew_cask - fixed issue where a cask with ``@`` in the name is incorrectly reported as invalid (https://github.com/ansible-collections/community.general/issues/733).
```

---

Thanks for the feedback @Andersson007 and @felixfontein !